docs/backup_restore.md

```diff
@@ -0,0 +1,52 @@
+In this section, you'll learn how to create backups of the rke2 cluster data and to restore the cluster from backup.
+
```

Comment:
Add a note here about the path prefix `/var/lib/rancher/rke2` being configurable by the `--data-dir` flag. If folks change the flag they will need to correct the path.

The way it reads now, folks are going to go looking for a /server/ path in the root of the filesystem.